_includes/cockroachcloud/create-a-sql-user.md

```diff
@@ -0,0 +1,17 @@
+{% include cockroachcloud/cockroachcloud-ask-admin.md %}
+
+1. Navigate to your cluster's **SQL Users** page.
+
+    <img src="{{ 'images/v19.2/cockroachcloud/sql-users.png' | relative_url }}" alt="SQL users" style="border:1px solid #eee;max-width:100%" />
+
+2. Click the **Add User** button in the top right corner.
+
+    The **Add User** modal displays.
```

Comment:
I think "dialog" is a more common term than "modal" - AIUI "modal" is an adjective that we are noun-ifying - it refers to a "modal dialog".

My (non-binding) opinion is that we should replace "modal" with "dialog" everywhere